python/ray/tune/ray_trial_executor.py

```diff
@@ -425,7 +533,8 @@ def start_trial(self, trial, checkpoint=None, train=True) -> bool:
             train (bool): Whether or not to start training.
 
         Returns:
-            True if trial was started successfully, False otherwise.
+            True if the remote runner has been started. False if trial was
+            not started (e.g. because of lacking resources/pending PG).
```

Comment:
```suggestion
            True if the remote runner has been started. False if trial was
                not started (e.g. because of lacking resources/pending PG).
```